zipline/data/data_portal.py

```diff
@@ -703,10 +697,17 @@ def _get_history_daily_window_data(
 
             return daily_data
 
-    def _handle_history_out_of_bounds(self, bar_count):
+    def _handle_minute_history_out_of_bounds(self, bar_count):
+        first_trading_minute_loc = (
```

Comment:
Can we save this calculated value somewhere so that we don't have to do it again later in the simulation?